corehq/apps/app_manager/tests/test_schedule.py

```diff
@@ -0,0 +1,408 @@
+from mock import patch
+import copy
+import base64
+from django.test import SimpleTestCase
+from corehq.apps.app_manager.models import (
+    DetailColumn,
+    Application,
+    FormSchedule,
+    ScheduleVisit,
+    SchedulePhase,
+    SchedulePhaseForm,
+    FormActionCondition,
+)
+from corehq.apps.app_manager.exceptions import ScheduleError
+from corehq.apps.app_manager.tests.util import TestFileMixin
+
+
+class ScheduleTest(SimpleTestCase, TestFileMixin):
+    file_path = ('data', 'suite')
+
+    def setUp(self):
+        self.is_usercase_in_use_patch = patch('corehq.apps.app_manager.models.is_usercase_in_use')
+        self.is_usercase_in_use_mock = self.is_usercase_in_use_patch.start()
+        self.is_usercase_in_use_mock.return_value = True
```

Comment:
Minor point, but I believe if you decorate the class with this patch, it'll be applied to each method that begins with `"test"`.  I'm not sure about specifying a return value in the decorator though, but I think something like this would work:
```python
@patch('corehq.apps.app_manager.models.is_usercase_in_use', lambda: True)
```